_data/toc.yaml

```diff
@@ -1435,8 +1435,10 @@ manuals:
       title: Install on Ubuntu
     - path: /engine/install/binaries/
       title: Install binaries
-  - path: /engine/install/linux-postinstall/
-    title: Optional post-installation steps
+    - path: /engine/install/linux-postinstall/
+      title: Post-installation steps
+  - path: /engine/troubleshoot/
+    title: Troubleshoot
```

Comment:
I wonder if it makes more sense to put this in the Install section and rename to something like 'Installation troubleshooting'. 
As it is, it makes me think I'm going to get troubleshooting topics for everything relating to Docker Engine, not just installing.